package.json

```diff
@@ -38,17 +38,17 @@
       },
       {
         "command": "vscode-flowr.slice.position",
-        "title": "Toggle Slice at the Current Position",
+        "title": "Continuously Slice at Current Position",
         "category": "flowR"
       },
       {
         "command": "vscode-flowr.slice.follow.cursor",
-        "title": "Toggle Slice Following the Cursor",
+        "title": "Continuously Slice at Cursor",
```

Comment:
I liked the idea, but i think the "toggle" part should remain to indicate that it allows to disable the feature as well!